web/source/osk/visualKeyboard.ts

```diff
@@ -175,8 +202,8 @@ namespace com.keyman.osk {
       }
     }
 
-    getKeyWidth(osk: VisualKeyboard): number {
-      let units = this.objectUnits(osk.isStatic);
+    getKeyWidth(vkbd: VisualKeyboard): number {
```

Comment:
You've renamed `osk` to `vkbd` here but not in other places

Reply:
This method does have other related changes (particularly, line 206), so I was fine altering it here.  The cases I didn't alter were generally because there were no other changes to those methods.

Admittedly, I wouldn't mind changing those too.  Just felt it was a bit out of scope.